src/app-gocardless/banks/lhv-lhvbee22.js

```diff
@@ -0,0 +1,42 @@
+import d from 'date-fns';
+
+import Fallback from './integration-bank.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['LHV_LHVBEE22'],
+
+  accessValidForDays: 90,
+
```

Comment:
This can be removed now as it's no longer used
```suggestion
```

Reply:
Removed in https://github.com/actualbudget/actual-server/pull/542/commits/7275d1f28210daad8ff18347cfe09e1ba5cfb3c4